homeassistant/components/unifiled/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "unifiled",
+  "name": "Unifi LED",
+  "documentation": "https://www.home-assistant.io/components/unifiled",
```

Comment:
Change `components` to `integrations` in the url.

Reply:
I have changed the url.